src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -2553,6 +2554,28 @@ public void sendEquipmentChange(@NotNull LivingEntity entity, @NotNull Equipment
 		// Pretend the packet gets sent.
 	}
 
+	@Override
+	public boolean isOp()
+	{
+		return this.op;
```

Comment:
This should get the status from the PlayerList, otherwise they can become desynced.